### ISSUE TITLE
Execution Failure with "npm run serve" - Cannot Find Module "postcss"

### ISSUE THREAD

**Issue by macZil** [vuejs/vue-cli#3990]
### Version
3.7.0

### Reproduction link
[https://github.com/macZil/simple-app](https://github.com/macZil/simple-app)

### Environment info
```
System:
    OS: Windows 10
    CPU: (12) x64 Intel(R) Core(TM) i7 CPU       X 980  @ 3.33GHz
  Binaries:
    Node: 10.15.3 - C:\Program Files\nodejs\node.EXE
    Yarn: Not Found
    npm: 6.9.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: 42.17134.1.0
  npmGlobalPackages:
    @vue/cli: Not Found
```

### Steps to reproduce
1. Create a new project with *vue create myApp*. 
2. Selected *default* preset.
2. *cd* into the project directory. 
3. Run *npm run serve* => Error:

```
"Module build failed (from ./node_modules/vue-loader/lib/index.js):
Error: Cannot find module 'postcss'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (H:\Projects\Coding\Vue\U_V2TCG\simple\node_modules\@vue\component-compiler-utils\dist\compileStyle.js:6:17)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (H:\Projects\Coding\Vue\U_V2TCG\simple\node_modules\@vue\component-compiler-utils\dist\index.js:7:24)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)

 @ ./src/main.js 6:0-28 10:13-16
 @ multi (webpack)-dev-server/client?http://192.168.8.149:8080/sockjs-node (webpack)/hot/dev-server.js ./src/main.js
```

### What is expected?
Expected CLI to start local dev server

### What is actually happening?
Getting this error whenever I try to run *npm run serve*

---
I've uninstalled and reinstalled the @vue/cli as well as tried deleting and recreating the project directory. 
I have done no customization to this project. It should work as it has every time in the past.

**Comment by sodatea** [comment]
Can't reproduce with the provided reproduction repository.

**Comment by macZil** [comment]
Thanks for trying @sodatea . After deleting and reinstalling the "node_modules" folder of an older project and running the local server I got the same error. 

I couldn't find any help on the net related to my issue but I remembered installing the "@vue/cli-service-global" a short while ago. I uninstalled that as well as uninstalled and reinstalled Node and the Vue CLI then created a new project and it all seems to be working now. Didn't pinpoint whether it was vue-service-global or Node, or both (as I removed them at the same time), but it's working again!
